src/lib/pages/frontsite/Documents/DocumentDetails/LoanRequestForm/LoanRequestForm.js

```diff
@@ -24,11 +24,13 @@ class LoanRequestForm extends Component {
       invenioConfig.CIRCULATION.deliveryMethods
     );
     this.deliveryMethods = this.withDeliveryMethod
-      ? Object.keys(invenioConfig.CIRCULATION.deliveryMethods).map((key) => ({
-          key: key,
-          value: key,
-          text: invenioConfig.CIRCULATION.deliveryMethods[key].text,
-        }))
+      ? Object.keys(invenioConfig.CIRCULATION.deliveryMethods)
+          .filter((key) => key !== 'SELF-CHECKOUT')
```

Comment:
Prevents from printing 'Self-checkout' in here:
<img width="1170" alt="Screenshot 2024-11-07 at 15 31 07" src="https://github.com/user-attachments/assets/34f1ee05-e4ef-41df-b320-42064061f4b7">

Reply:
Is this also taken care of in the backend if we do it via the API?

Reply:
> Is this also taken care of in the backend if we do it via the API?

Not really, it is a bit complicated to handle in the backend. I would not filter it out here, as I would use a different config var.
And for the backend, someone will have to know that SELF-CHECKOUT can be pushed. And... not very harmful, it is just a text (no different behaviour).